examples/mugen/retrieval/video_clip.py

```diff
@@ -75,26 +78,41 @@ class VideoEncoder(nn.Module):
     Adapted from MUGEN's video encoder
         (https://github.com/mugen-org/MUGEN_baseline/blob/main/lib/models/videoclip/modules.py)
 
+    Attributes:
+        model (nn.Module): Module extracted from :class:`~torchvision.models.video.S3D`.
+            Code reference: https://github.com/pytorch/vision/blob/main/torchvision/models/video/s3d.py
+        return_node_name (str): Name of the last layer extracted from S3D.
+        out_dim (int): Output dimension of VideoEncoder.
+
     Inputs:
         x (Tensor): batch of videos with dimensions (batch, channel, time, height, width)
             Size of ``channel`` dimension must be ``3``.
 
     """
 
-    def __init__(
-        self,
-    ):
+    def __init__(self, num_classes: int = 400) -> None:
```

Comment:
Since we only use as encoder, I think the `num_classes` here is not really used.
I think we can remove this param and just do: `base = S3D()` instead.